src/main/webapp/resources/js/components/samples/components/SampleInfo.tsx

```diff
@@ -13,8 +13,10 @@ import { formatDate } from "../../../utilities/date-utilities";
 const { Paragraph } = Typography;
 import moment from "moment";
 import { OntologyInput } from "../../ontology";
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
What is this here for?

Reply:
Just to stop the error displaying for `import { TAXONOMY } from "../../../apis/ontology/taxonomy";`:

TS2306: File '/home/deep/irida/src/main/webapp/resources/js/apis/ontology/taxonomy/index.js' is not a module.

Still something I need to look into but I'm assuming the error is because the file is a js file instead of a ts

Reply:
Why don't you just update the file then, it will be good for everyone?

Reply:
I'm going to. Just was getting rid of the sea of red in intellij.

Reply:
Updated in 445c58cec70fbfbbe6113426d30eb0caf860e90f